src/components/countdown.tsx

```diff
@@ -0,0 +1,66 @@
+import React, {useEffect, useState} from 'react';
+import styled from 'styled-components';
+
+const StyledCountdown = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: space-evenly;
+  align-items: flex-start;
+
+  padding: 16px;
+
+  width: 250px;
+  height: 101px;
+  left: 25px;
+  top: 0px;
```

Comment:
?

Reply:
The width will be 100% of the column that contains it.
Instead of setting a fixed height, set an aspect ratio instead.

Don't use left/top for spacing. Try to use `margin` instead.